docs/patrol/native-setup.mdx

```diff
@@ -55,13 +55,12 @@ features:
 
 ### Install patrol_cli
 
-Patrol CLI (command-line interface) is a small program that makes it easier to
-run Flutter integration tests using Patrol's native automation feature. It
-handles the complexity of native iOS and Android testing tools itself, so you
-can focus on writing your tests instead of learning yet another
-platform-specific tool.
+Patrol CLI (command-line interface) is a small program that enables running
+Patrol UI tests. It handles the complexity of native iOS and Android testing
+tools itself, so you can focus on writing your tests instead of learning yet
+another platform-specific tool.
 
-1. Install `patrol` executable:
+1. Install `patrol_cli` executable:
 
    ```
    dart pub global activate patrol_cli
```

Comment:
I think it's OK. We limit breaking changes to Patrol CLI to a minimum. Also, it may not be the best argument, but I feel like it's common in the community.

I'd lean towards adding `<Info>` saying "Hey, you can also install locally, like this: <--->"